lib/ansible/modules/cloud/amazon/iam_cert.py

```diff
@@ -269,10 +276,21 @@ def main():
     cert_chain = module.params.get('cert_chain')
     dup_ok = module.params.get('dup_ok')
     if state == 'present':
-        cert = open(module.params.get('cert'), 'r').read().rstrip()
-        key = open(module.params.get('key'), 'r').read().rstrip()
+        try:
+            cert = open(module.params.get('cert'), 'r').read().rstrip()
+        except IOError:
```

Comment:
I'm going to add a commit to do `if os.path.exists(module.params.get('cert')):` instead of try/except.